build.gradle.kts

```diff
@@ -9,13 +9,53 @@ plugins {
     id("io.spring.dependency-management") version "1.0.11.RELEASE" apply false
     id("org.sonarqube") version "3.3"
     id("org.jetbrains.kotlin.jvm") apply false
+    id("com.diffplug.spotless") version "6.11.0"
     `maven-publish`
     signing
 }
 
 val projectTitle: String by project
 val sonarLinksCi: String by project
 
+configure(subprojects.filter { it.name != "kotlin-checks-test-sources"}) {
+    apply(plugin = "com.diffplug.spotless")
+
+    configure<com.diffplug.gradle.spotless.SpotlessExtension> {
+
+        lineEndings = com.diffplug.spotless.LineEnding.UNIX
+
+        fun SourceSet.findSourceFilesToTarget() = allJava.srcDirs.flatMap { srcDir ->
+            project.fileTree(srcDir).filter { file ->
+                file.name.endsWith(".kt") || (file.name.endsWith(".java") && file.name != "package-info.java")
+            }
+        }
+
+        kotlin {
+            // ktlint()
```

Comment:
Can we remove this or add a comment explaining why there might have been a good reason to call `ktlint`?

Reply:
```suggestion
```

Reply:
I think we wanted to use ktlint at some point but then decided not to, as we were not happy with the result.

I think it's okay to drop the line and re-launch the discussion in the future if we want to use it or not.